### PR TITLE
chore: update cl for release

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Updates changelog for release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+<a name="10.3.0"></a>
+
+## [10.3.0](https://github.com/commercetools/ui-kit/compare/v10.2.1...v10.3.0) (2019-10-08)
+
+### Features
+
+This release represents the first release that contains the new `RichTextInput`. Keep in mind this component is still in beta and may be subject to upcoming breaking changes. We will not consider breaking changes to `RichTextInput` to be breaking changes to UI-Kit until the component is out of beta status.
```
Comment (montezume) [inline CHANGELOG.md]: WDYT @jonnybel ?

Comment (tdeekens) [reply CHANGELOG.md]: Does storybook have a beta-badge plugin or sth?

**Comment by montezume** [comment]
@tdeekens 
Screen Shot 2019-10-08 at 2 13 25 PM

I didn't tag you because of that 🤔

File: CHANGELOG.md (hunk 1 above)
Comment (jonnybel) [reply CHANGELOG.md]: I like the approach 👍

**Pull request merged**